### PR TITLE
project description and overview

### PR DESCRIPTION
**Pull request by grantmiller**
What I Did
------------
changed the description of what ship is

How I Did it
------------
wrote some copy

How to verify it
------------
read it

Description for the Changelog
------------
improved readme

Picture of a Boat (not required but encouraged)
------------

### CODE REVIEW

**Comment by fshot** [comment]
evoke ... should be invoke, yeah?  

"This is possible because of how the three operating modes of Ship **evoke**, store and apply Kustomizations made by the cluster operator."

File: README.md [hunk 1]
```diff
@@ -9,33 +9,36 @@ Ship
...
+Replicated Ship is an open source project by [Replicated](https://www.replicated.com) designed to extend Google’s [Kustomize](www.kustomize.io) project in a way that can reduce the operational overhead of maintaining 3rd party applications (open source or proprietary) deployed to a [Kubernetes](https://kubernetes.io) cluster.
 
-Ship includes first-class support for [Helm](https://helm.sh/) charts, and automates the "last-mile" of custom configuration via [Kustomize](https://kustomize.io/).
+The initial release of Replicated Ship exposes the power of Kustomize as an advanced custom configuration management tool for [Helm charts](https://www.github.com/helm/charts), Kubernetes manifests and [Knative](https://github.com/knative/) applications. With Ship, cluster operators can automatically stay in sync with upstream changes (ie. via automated pull requests or another form of automation) while preserving their local, custom configurations and extensions (add, deletes and edits) without git merge conflicts. This is possible because of how the three operating modes of Ship evoke, store and apply Kustomizations made by the cluster operator.
```
Comment (laverya) [inline README.md]: Should we do one sentence per line here? It makes the diffs cleaner, and markdown merges adjacent paragraphs without an empty line separating them

Comment (grantmiller) [reply README.md]: good idea

File: README.md [hunk 2]
```diff
@@ -9,33 +9,38 @@ Ship
...
-## ship watch
-Polls an upstream source, blocking until any change has been published.  Use for:
-- Triggering creation of pull requests in a CI pipeline, so that third party updates can be manually reviewed, and then automatically deployed once merged
+# Features
+Ship is designed to provide first-time configuration UI and/or be used headless in a CI/CD pipeline to automate deployment of third party applications.
+
+- Web based "admin console" provides initial configuration of [Helm](https://helm.sh/) values and creates [Kustomize](https://kustomize.io/) overlays
+- Headless mode supports automated pipelines
+- Merge [Helm](https://helm.sh/) charts with override values and apply custom overlays with [Kustomize](https://kustomize.io/) to avoid merge conflicts when upstream or local values are changed
+- Observe upstream repos for changes & therefore triggering update processes.
```
Comment (dexhorthy) [inline README.md]: This sentence feels a little off

**Pull request merged**